services/user/CommonApi/common/packages/plugin-tester/ui/src/components/editors/RawParameterEditor.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from "react";
+
+interface RawParameterEditorProps {
+  values: Record<string, unknown>;
+  onChange: (values: Record<string, unknown>) => void;
+}
+
+export const RawParameterEditor: FC<RawParameterEditorProps> = ({
+  values,
+  onChange,
+}) => {
```

Comment:
Best practice for functional components is what you're doing elsewhere without `FC`:

`export const RawParameterEditor = ({ values, onChange }: RawParameterEditorProps) => { ... }`

Reply:
Before React 18, `React.FC` was not recommended, for various reasons. It's technically fine now. More a style preference than anything else.